jenkins_build.sh

```diff
@@ -0,0 +1,47 @@
+set -e
+
```

Comment:
Please add some comments to document which environment variables must be set by the caller and what each means.